scanner/Makefile

```diff
@@ -20,7 +20,7 @@ endif
 endif
 
 ifeq ($(TAG),)
-	TAG = $(shell $(MAKE) -C ../ tag)
+	TAG = $(shell $(MAKE) -C ../ --quiet --no-print-directory tag)
```

Comment:
You might want to use `:=` instead of `=`.

Reply:
https://github.com/stackrox/stackrox/pull/7270